lib/LaTeXML/Core/Token.pm

```diff
@@ -321,12 +317,13 @@ sub with_dont_expand {
 # or undef if it isn't marked as such.
 sub get_dont_expand {
   my ($self) = @_;
-  return ($$self[1] == CC_NOEXPAND1 ? $$self[2]->get_dont_expand : $$self[2]); }
+  return $$self[2]; }
 
 sub without_dont_expand {
   my ($self) = @_;
-  # Remove dont_expand flag, but keep NOEXPAND1 wrapper
-  return (($$self[1] == CC_NOEXPAND1) ? ($$self[2][2] ? T_NOEXPAND1($$self[2][2]) : $self) : ($$self[2] || $self)); }
+  # Remove dont_expand flag, remove NOEXPAND1 wrapper
```

Comment:
We can throw it all away! `noexpand1` is even less sticky than the Real noexpand, so definitely can go.

Reply:
Ah, missed that! You know how hard it was to get all that wrong-right?